js/fixToc.js

```diff
@@ -0,0 +1,65 @@
+document.addEventListener("DOMContentLoaded", () => {
+  const SCROLL_OFFSET = 80; // Adjust based on your header height
```

Comment:
Should we be fetching the header height? Instead of hard-coding it? What if we change the header size (like if we were to go to two rows like Wormhole) down the road and this breaks without us even being aware of it